### DISCUSSION TITLE
Mullvad Browser - Not recognizing the installed companion app

### DISCUSSION THREAD

**Discussion by lamyergeier** [aclap-dev/video-downloadhelper#848] [Help and Q&A]
# Mullvad Browser - Not recognizing the installed companion app 

![image](https://github.com/aclap-dev/video-downloadhelper/assets/42092626/8fbb5b34-d294-4f7f-9886-4e37c63f3739)

# Info

    The browser you're using (be as specific as possible): Mullvad Browser 13.0.15 (based on Firefox 115.11.0esr)
    OS (Linux/Windows/Mac) & OS version: Fedora 40
    OS 64bits or 32bits: 64
    On Linux, your sandbox mechanism (if any): No. Downloaded the tar from GitHub and extracted the browser

## $ ~/.local/share/vdhcoapp/vdhcoapp --info

```json
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/home/lamy/.local/share/vdhcoapp/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/lamy",
  "converterBinary": "/home/lamy/.local/share/vdhcoapp/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360"
}
```

### Addon & CoApp version and details

Addon: 9.0.2.3
CoApp: 2.0.19

**Comment by Mattmjs** [comment]
Did you refer to the wiki: https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized#linux-all
Usually an extra command needs to be run to fully register the app.

**Reply by lamyergeier** [reply]
I have read that before raising this issue, and executed that command. It is successfully recognised in Firefox and Chrome but not on Mullvad Browser

**Reply by LEASoCal** [reply]
did not work for me either

**Comment by lamyergeier** [comment]
![image](https://github.com/aclap-dev/video-downloadhelper/assets/42092626/34ad588b-b22e-4715-9f86-a6084c3214b9)

**Comment by Mattmjs** [comment]
You might have to search other discussions for Tor Browser. Since there was issues for those using Tor, there ended up being
a solution which could be relevant to Mullvad Browser.

**Reply by GrampaWildWilly** [reply]
Actually, use "recognize coapp" as your search key.  I just gave similar advice over here: #854.  You might want to read what I had to say over there.

**Comment by LEASoCal** [comment]
I have the same issue and none of the links make any sense to me.
